RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -19,22 +19,19 @@ public PassParameterByReferenceQuickFix(ParserRuleContext context, QualifiedSele
 
         public override void Fix()
         {
-            var parameter = Context.GetText();
+            string pattern = "^\\s*" + Tokens.ByVal + "(\\s+)";
+            string rgxReplacement = Tokens.ByRef + "$1";
+            Regex rgx = new Regex(pattern);
 
-            var parts = parameter.Split(new char[]{' '},2);
-            if (1 != parts.GetUpperBound(0))
-            {
-                return;
-            }
-            parts[0] = parts[0].Replace(Tokens.ByVal, Tokens.ByRef);
-            var newContent = parts[0] + " " + parts[1];
+            var parameter = Context.GetText();
```

Comment:
OK - I've updated to use a strategy similar to comintern's example.  Just for my future reference - I used regular expressions in the prior solution because I found that they were used frequently in the current code base.  Am I to understand that the preference is to use approaches other than regular expressions going forward?  Thanks.